core-rust/mesh-api-server/src/mesh_api/errors.rs

```diff
@@ -51,6 +51,8 @@ pub(crate) enum ApiError {
     InvalidBlockIdentifier,
     #[strum(serialize = "Submit transaction error")]
     SubmitTransactionError,
+    #[strum(serialize = "State history not available")]
```

Comment:
Is this the API error message... I wonder if we wish to say a little more - e.g. advise node runners that it can be enabled with XXX configuration option, like I believe we do in the Core API?

Reply:
In fact this error is used when some `StateHistoryError` occurs when getting state history (the details of the error are attached to the `details_message`)
I will rename it to `GetStateHistoryError`